src/Compilers/CSharp/Portable/Emitter/Model/SymbolAdapter.cs

```diff
@@ -126,5 +126,23 @@ internal IEnumerable<CSharpAttributeData> GetCustomAttributesToEmit(ModuleCompil
                 }
             }
         }
+
+        internal Cci.TypeReferenceWithAttributes GetTypeRefWithAttributes(TypeSymbol type, Cci.ITypeReference typeRef)
```

Comment:
Eh. I don't really see anything else using the type ref attributes. If we have another significant use I'd be fine with pulling it out.